index.html

```diff
@@ -52,14 +52,14 @@ <h2 id="main-h2">Private Digital Currency</h2>
                         </div>
                         <div class="col-lg-6 col-md-6 col-sm-8 col-xs-12 why-text">
                             <h3>Monero is private.</h3>
-                            <p>The power of the blockchain usually increases security at the cost of privacy, but with Monero's sophisticated privacy-centric technology, you get all of the security benefits of the blockchain without any of the privacy trade-offs.</p>
+                            <p>Security of the blockchain normally increases at the cost of privacy. Monero is based on the Cryptonote application layer protocol, which allows for the security benefits of blockchain technology without any of the privacy trade-offs</p>
```

Comment:
> Monero's sophisticated privacy-centric technology
I think you're on the right track that this line needs to be updated. (It's a bit of puffery IMO.)

Maybe: `Blockchain technology usually increases security at the cost of privacy, but because Monero is built with ring-signatures, it has hidden amounts, destinations, and origins of transactions. That means it gains all the benefits of having a public blockchain, without any of the normal privacy concessions.'